spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -732,6 +744,7 @@ The following built-in annotations define a `Literal` static nested class to sup
 * `jakarta.enterprise.inject.Any`
 * `jakarta.enterprise.inject.Default`
 * `jakarta.enterprise.inject.Specializes`
+// TODO move to Full?
```

Comment:
I noticed these as well. IMO if the user gets to consume the exact same CDI API jar, then there is little reason to extract this from spec as those literals are simply present, even though their respective annotations are ignored by Lite.

Or you could just separate these two and add them right below, stating that these are related to CDI Full?